README.md

```diff
@@ -249,6 +250,19 @@ kubeaudit np
 WARN[0000] Default allow mode on test/testing
 ```
 
+<a name="resources" />
+
+### Audit resource quotas
+
+It checks that every resources should have a CPU and memory limts. See [Kubernetes Resource Quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/)
+for more information:
+
+```sh
+kubeaudit -l quota
+ERRO[0000] CPU quota not set, please set it!
```

Comment:
I think these should be `WARN`ings and not `ERRO`rs. I am pulling in @JonPulsifer for some feedback on this

Reply:
Maybe the Warn level is better in this case, I will change it.